static/app/views/dashboards/widgets/tableWidget/tableWidgetVisualization.tsx

```diff
@@ -75,12 +77,21 @@ interface TableWidgetVisualizationProps {
    * @param meta The full table metadata
    */
   makeBaggage?: BaggageMaker;
+  /**
+   * A callback function that is invoked after a user resizes a column. If omitted, resizing will update the width parameters in the URL
+   * @param widths an array of numbers containing the widths by column order
+   */
+  onChangeResizeColumn?: (widths: number[]) => void;
+
```

Comment:
This is a bit strange, I think!

First of all, `onChangeResizeColumn` is a bit of an odd name since "resize" makes "change" redundant. Maybe `onResizeColumn` would be better?

Second, I think passing `widths` to the callback is a bit odd. Looking around the codebase, most usages of `GridEditable` take the resized column and merge it back into a `columns` array, rather than keeping separate state for the width. I think this component should follow that pattern, and call `onResizeColumns` with `columns: TabularColumn[]`, so parent components can save that information directly in React state.